libraries/ms-common/src/stm32f0xx/spi.c

```diff
@@ -11,14 +11,10 @@ typedef struct {
 
 static SPIPortData s_port[SPI_MCU_NUM_PORTS] = {
   {
-    .rcc_cmd = RCC_APB2PeriphClockCmd,
-    .periph = RCC_APB2Periph_SPI1,
-    .base = SPI1
+      .rcc_cmd = RCC_APB2PeriphClockCmd, .periph = RCC_APB2Periph_SPI1, .base = SPI1,
```

Comment:
Ah, I see. So if we don't have a comma, will it pack the initializer like a function call or multi-line?